scripts/test/run_selenium_tests_dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# cd to root of repo
+cd "$( dirname "${BASH_SOURCE[0]}" )"/../../
+
+if [[ ! -e "webpack-stats.json" ]]
+then
+    echo "Please start the webpack dev server before running this script."
+    exit 1
+fi
+
+# TODO: can we detect this somehow?
+if [[ -z "$WEBPACK_DEV_SERVER_HOST" ]]
+then
+    echo "Please set WEBPACK_DEV_SERVER_HOST to the IP address of your webpack dev server, omitting the port number."
+    exit 1
+fi
```

Comment:
it would be cool if we could call `docker inspect`, parse the JSON, and figure out the IP automatically - do you think that would be too complicated? I imagine it would be tricky to deal with the Mac environment as well.